### DISCUSSION TITLE
Is there a way to exclude some payer identification types?

### DISCUSSION THREAD

**Discussion by artmadeit** [mercadopago/sdk-js#139] [Q&A] [answered]
Similar to exclude payment methods:

https://www.mercadopago.com.pe/developers/es/docs/checkout-bricks/card-payment-brick/additional-customization/configure-payment-methods

Is there a way to exclude some payer identification types?
for example on Perú there are these identification types: DNI, RUC, C.E., Otros
but what if I want to exclude C.E. and Otros from being displayed on the payer identification type dropdown

![image](https://user-images.githubusercontent.com/18428325/222926546-04cf7f68-a007-483f-a529-4f049b89a890.png)

**Comment by icaldana** [comment, marked as answer]
Hello @artmadeit!

The payer identification types aren't customizable, but I would like to undersdand why you need it.
Because by excluding identification types, we would be will reducing th  audience able to use the checkout, and we assume this is not what you desire.

In case this demand is caused by https://github.com/mercadopago/sdk-js/issues/138, we suggest you to please hold on while we investigate and fix it. How does that sound?

Thank you.

**Reply by artmadeit** [reply]
Yes, the reason is to reduce the audience. Here on Perú sometimes you want to only enable payments for some types of customers. For example a B2B application only allows customers that are enterprises ( so the only valid option for identification type should be RUC). Other example, is an app that just want to sell their products to people of their country so Carnet de extranjeria and Otros option should be hidden

**Reply by icaldana** [reply]
> Yes, the reason is to reduce the audience. Here on Perú sometimes you want to only enable payments for some types of customers. For example a B2B application only allows customers that are enterprises ( so the only valid option for identification type should be RUC). Other example, is an app that just want to sell their products to people of their country so Carnet de extranjeria and Otros option should be hidden

Got it. I brought your need to our backlog, and we will carefully consider it, but sadly for now there is no native support for this filter.

**Reply by artmadeit** [reply]
Thanks..no problem let me know when that will be done. It doesn't matter if it is implemented on a week or several months. Thanks 👍
